docs/index.html

```diff
@@ -1690,7 +1700,9 @@ <h1 class="display-4 theme-content-element-title">Tooltip</h1>
 		<span class="theme-content-element-source-label">Source:</span>
 		<span class="theme-content-element-source-path">tooltip.md</span>
 	</span>
-	<div class="theme-content-element-description"><p>A tooltip can give context to a given icon/piece of text. Tooltips are usually hidden and become visible after an action or when hovering.</p>
+	<div class="theme-content-element-description"><p>A tooltip can give context to a given icon/piece of text. Tooltips are usually hidden and become visible on hovering, focus or on touch.</p>
+<h3>✅ When to use:</h3><p>Tooltips can be used to provide the user with brief and useful information related to a feature.</p>
+<h3>⛔ When not to use:</h3><p>Tooltips should not contain necessary and critical information needed by the user to complete a task.</p>
```

Comment:
De fungerer ikke på mobil, bør ha med det på not to use

Reply:
Sure, plocka bort det

Reply:
"Tooltips should not contain necessary and critical information needed by the user to complete a task. Tooltips does not work on mobile devices as they use the hover class, which only works with a mouse"